app/migrations.go

```diff
@@ -479,6 +480,33 @@ func (s *Server) doFirstAdminSetupCompleteMigration() {
 	}
 }
 
+func (s *Server) doRemainingSchemaMigrations() {
+	// If the migration is already marked as completed, don't do it again.
+	if _, err := s.Store.System().GetByName(remainingSchemaMigrationsKey); err == nil {
+		return
+	}
+
+	if teams, err := s.Store.Team().GetByEmptyInviteID(); err != nil {
```

Comment:
Since there's only "one remaining", should we name this function appropriately? Or is the idea to extend it in the future?

Also, it's not clear to me why this is at the application layer, vs. a migration script -- could we run the equivalent of `model.NewId()` in SQL?

Reply:
> could we run the equivalent of model.NewId() in SQL?

I don't think it's impossible, but I reckoned that it might be simpler to do it in the app layer.